p2pserver/p2pserver.go

```diff
@@ -141,6 +146,34 @@ func (this *P2PServer) GetNeighborAddrs() []common.PeerAddr {
 	return this.network.GetNeighborAddrs()
 }
 
+// GetPubKey return local public key
+func (this *P2PServer) GetPubKey() keypair.PublicKey {
+	return this.network.GetPubKey()
```

Comment:
there'll probably  no publickey on p2p server.
what this function used for?

Reply:
It is to get local public key to sign the data before broadcasting message. Even if no publickey on p2p server in the future, we need to know where to sign the data.

Reply:
get it from local account.

Reply:
Updated.